doc/files/package.json.md

```diff
@@ -202,6 +202,8 @@ Conversely, some files are always ignored:
 * `npm-debug.log`
 * `.npmrc`
 * `node_modules`
+* `.npmrc`
```

Comment:
Yeah, you are right. As I see this page last update was made on January 29, 2016. Should be deployed